iquip/apps/dataviewer.py

```diff
@@ -73,12 +77,25 @@ def setData(self, data: np.ndarray, axes: Sequence[AxisInfo]):
             if size != len(info.values):
                 raise ValueError(f"Size mismatch in {info}: expected {size} values")
 
+    def nearestDataPoint(
+        self, scenePos: pg.Point, tolerance: Optional[float] = None,  # pylint: disable=unused-argument
+    ) -> Optional[Tuple[int, ...]]:
+        """Returns the index of the nearest data point.
+        
+        Args:
+            scenePos: Scene position coordinates.
+            tolerance: Maximum Euclidean distance in the scene coordinate,
```

Comment:
This is just a question! Is the unit of `tolerance` a pixel?

Reply:
I think so..! "Scene position" seems to be in pixel.